### PR TITLE
bump commander to 0.28.2

### PR DESCRIPTION
**Pull request by andriisoldatenko**
Bump commander to include correct airflow-chart version (1.3.0):
```
docker run -it --entrypoint=sh quay.io/astronomer/ap-commander:0.28.2
Unable to find image 'quay.io/astronomer/ap-commander:0.28.2' locally
0.28.2: Pulling from astronomer/ap-commander
59bf1c3509f3: Already exists
0a0d3fa14c98: Pull complete
4f037c6f701e: Pull complete
4b513c630b4b: Pull complete
59f2014a4b8b: Pull complete
9675e2e06a6d: Pull complete
0bca3117399e: Pull complete
5b27feb7eec2: Pull complete
ee646cde4aa3: Pull complete
7c330cab5f96: Pull complete
c6f70f72619a: Pull complete
Digest: sha256:59e9478c3dba052be5a5350ab34fec22e643aed2940d9eda5fe736c171373110
Status: Downloaded newer image for quay.io/astronomer/ap-commander:0.28.2
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
/app $ cat
astronomer-ee/  commander       stable/
/app $ ls -la astronomer-ee/airflow-1.3.0.tgz
-rw-r--r-- 1 commander commander 180417 Mar 15 17:23 astronomer-ee/airflow-1.3.0.tgz
/app $
```

resolves https://github.com/astronomer/issues/issues/4390

### CODE REVIEW

File: charts/astronomer/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: astronomer
-version: 0.28.0
+version: 0.28.1
```
Comment (vishwas-astro) [inline charts/astronomer/Chart.yaml:3]: We are already at 0.28.2 now. This should be 0.28.3

File: Chart.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: astronomer
-version: 0.28.0
-appVersion: 0.28.0
+version: 0.28.1
+appVersion: 0.28.1
 description: Helm chart to deploy the Astronomer Platform
```
Comment (vishwas-astro) [inline Chart.yaml:5]: We are already at 0.28.2 now. This should be 0.28.3

File: charts/astronomer/Chart.yaml (hunk 1 above)
Comment (danielhoherd) [reply charts/astronomer/Chart.yaml:3]:
```suggestion
version: 0.28.3
```

File: Chart.yaml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: astronomer
-version: 0.28.0
-appVersion: 0.28.0
+version: 0.28.1
+appVersion: 0.28.1
```
Comment (danielhoherd) [inline Chart.yaml:4]:
```suggestion
version: 0.28.3
appVersion: 0.28.3
```

**Review by danielhoherd — CHANGES_REQUESTED** [review]
Looks good once the version changes that @vishwas-astro pointed out are merged.

File: Chart.yaml (hunk 2 above)
Comment (bote795) [reply Chart.yaml:5]: should this be done in main or just release branch?

**Comment by bote795** [comment]
we are also missing the issue this fixes

**Comment by bote795** [comment]
we are also missing the issue this fixes

**Review by vishwas-astro — APPROVED** [review]

**Comment by andriisoldatenko** [comment]
@bote795 resolves https://github.com/astronomer/issues/issues/4390

**Review by bote795 — APPROVED** [review]

File: Chart.yaml (hunk 2 above)
Comment (andriisoldatenko) [reply Chart.yaml:5]: @vishwas-astro can you please help here? I'm not sure that i understand @danielhoherd

**Pull request merged**